client/post-editor/media-modal/index.jsx

```diff
@@ -427,7 +443,7 @@ export class EditorMediaModal extends Component {
 		if ( this.state.source !== '' ) {
 			buttons.push( {
 				action: 'confirm',
-				label: this.props.translate( 'Copy to media library' ),
+				label: selectedItems.length > 1 ? this.props.translate( 'Copy to media library' ) : this.props.translate( 'Insert' ),
```

Comment:
AFAIK, the site icon modal gets its button name [from here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/site-icon-setting/index.jsx#L310), and the set featured image [from here](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/editor-featured-image/index.jsx#L90) - so this change doesn't affect them.

I've also tested and it works for me. Do you get this text on the site icon and featured image modals?